hash/megadriv.xml

```diff
@@ -32902,5 +32878,30 @@ This dump is either a bad dump or a wrongly patched one.
 		</part>
 	</software>
 
+	<!-- the Radica titles are handled as standalone sets in MAME, left here for reference -->
```

Comment:
Please don't leave commented crud - it just rots.

Reply:
it serves as an example of how you can still add them / load them with the softlist if you want, which I think is useful.